content/en/blog/falcosidekick-knative.md

```diff
@@ -0,0 +1,460 @@
+[INFO]  Verifying binary download
+[INFO]  Installing k3s to /usr/local/bin/k3s
+[INFO]  Creating /usr/local/bin/kubectl symlink to k3s
+[INFO]  Creating /usr/local/bin/crictl symlink to k3s
+[INFO]  Creating /usr/local/bin/ctr symlink to k3s
+[INFO]  Creating killall script /usr/local/bin/k3s-killall.sh
+[INFO]  Creating uninstall script /usr/local/bin/k3s-uninstall.sh
+[INFO]  env: Creating environment file /etc/systemd/system/k3s.service.env
+[INFO]  systemd: Creating service file /etc/systemd/system/k3s.service
+[INFO]  systemd: Enabling k3s unit
+Created symlink /etc/systemd/system/multi-user.target.wants/k3s.service → /etc/systemd/system/k3s.service.
+[INFO]  systemd: Starting k3s
+```
+
+Now we have a bare bones k3s cluster!
+
+### Install Knative
+
+To install the rest of Knative into k3s:
+
+```bash
+# Installs Knative Serving
+kubectl apply -f https://github.com/knative/serving/releases/download/v0.22.0/serving-crds.yaml
+kubectl wait --for=condition=Established --all crd
+kubectl apply -f https://github.com/knative/serving/releases/download/v0.22.0/serving-core.yaml
+kubectl apply -f https://github.com/knative/net-kourier/releases/download/v0.22.0/kourier.yaml
+kubectl patch configmap/config-network \
+  --namespace knative-serving \
+  --type merge \
+  --patch '{"data":{"ingress.class":"kourier.ingress.networking.knative.dev"}}'
+
+# Installs Knative Eventing
+kubectl apply -f https://github.com/knative/eventing/releases/download/v0.22.0/eventing-crds.yaml
+kubectl wait --for=condition=Established --all crd
+kubectl apply -f https://github.com/knative/eventing/releases/download/v0.22.0/eventing-core.yaml
+kubectl apply -f https://github.com/knative/eventing/releases/download/v0.22.0/in-memory-channel.yaml
+kubectl apply -f https://github.com/knative/eventing/releases/download/v0.22.0/mt-channel-broker.yaml
+
+# Creates a default Broker
+kubectl create -f - <<EOF
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  name: default
+  namespace: default
+EOF
+```
+
+See also
+[knative.dev install instructions](https://knative.dev/docs/install/any-kubernetes-cluster/)
+for installed these into your own cluster.
+
+### Falco/Falcosidekick/sidekick UI
+
+We'll use helm to install `Falco` ,`Falcosidekick` and `Falcosidekick UI`.
+
+First, add the falcosecurity `helm` repo:
+
+```shell
+helm repo add falcosecurity https://falcosecurity.github.io/charts
+```
+
+In a real project, you should get the whole chart with
+`helm pull falcosecurity/falco --untar` and then configure the `values.yaml`.
+For this tutorial, will try to keep thing as easy as possible and set configs
+directly by `helm install` command:
+
+```shell
+helm install falco falcosecurity/falco \
+  --create-namespace --namespace falco \
+  --set falcosidekick.enabled=true --set falcosidekick.webui.enabled=true \
+  --set falcosidekick.config.cloudevents.address=http://broker-ingress.knative-eventing.svc.cluster.local/default/default
```

Comment:
Might be worth explaining the address value is the endpoint to the default broker created above

Reply:
@n3wscott wdyt? this is the only comment which remains. After its resolution, I'm ready to approve.